core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java

```diff
@@ -520,7 +520,7 @@ private String transformOMAScore( String omaScore) {
         } else if( omaScore.equalsIgnoreCase("N") || omaScore.equalsIgnoreCase("neutral")) {
             return "N";
         } else if( omaScore.equalsIgnoreCase("[sent]")) {
-            return ExtendedMutationUtil.NOT_AVAILABLE; // TODO temp workaround for invalid sent values
+            return "NA"; // TODO temp workaround for invalid sent values
```

Comment:
Hi @pieterlukasse I'm not familiar with this code (@onursumer may have written it).  It does seem to be related to the mutation assessor.  @sheridancbio is still involved with the support of the site and he tells me that the getma.org reference is no longer active.  I think we still need to provide links to the site - either updating the code to support the new URL or finding another solution entirely.